<launch>
    <arg name="ns" default="PointCloudTo3dMesh"/>
    <arg name="exec_name" default="PointCloudTo3dMesh"/>

     <node pkg="tf"
        type="static_transform_publisher"
        name="kinect_broadcaster"
        args="0 0 0 0 0 0 world kinect2_rgb_optical_frame  100" />
     <node pkg="tf" 
        type="static_transform_publisher" 
        name="to_origin_broadcaster" 
        args="0 0 1.5 0 0 3.14 world tf_broadcaster 100" />
  <node pkg="rviz" 
        type="rviz" 
	name="rviz"/>


   
    <rosparam command="load" file="$(find point_cloud_to_3d_mesh)/launch/params/kinectTopics.yml" ns="$(arg ns)"/>

    <node name="$(arg exec_name)" pkg="point_cloud_to_3d_mesh" type="$(arg exec_name)"  output="screen" launch-prefix="gdb --args" >

    </node>
</launch>
